auditbeat/docs/howto/howto.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[howto-guides]]
+= How to
+
+[partintro]
+--
+Learn how to perform common {beatname_uc} configuration tasks.
+
+* <<{beatname_lc}-geoip>>
+* <<using-environ-vars>>
+* <<configuring-ingest-node>>
+* <<yaml-tips>>
+
+
+--
+
+include::{libbeat-dir}/shared-geoip.asciidoc[]
+
+:standalone:
+include::{libbeat-dir}/shared-env-vars.asciidoc[]
+:standalone!:
+
+include::{libbeat-dir}/shared-config-ingest.asciidoc[]
+
+:standalone:
+include::{libbeat-dir}/yaml.asciidoc[]
+:standalone!:
```

Comment:
This section doesn't include the `include::{docdir}/../docs/reload-configuration.asciidoc[]` file for auditbeat, which means the content doesn't appear in the documentation.

Reply:
Yah sorry, this PR isn't quite ready for a close review (hence my to-do item "Do diff across all books to verify that nothing was dropped"). I just wanted to get the files pushed before the weekend. I still need to do some final changes. :-)

Reply:
Gotcha! Sorry bout that.